shell/packages/sandstorm-accounts-ui/login_buttons.html

```diff
@@ -1,7 +1,7 @@
 <template name="loginButtons">
   {{!-- Data context is an instance of AccountsUi. --}}
   {{#if isDemoUser}}
-    <a class="{{#if expiringSoon}}expiring-soon{{/if}}">Demo {{demoTimeLeft}} ▾</a>
+    <a>Create Account ▾</a>
```

Comment:
That's pretty sensible. +0 at least. (I'd prefer "Create free account" but we can do an experiment to fine-tune that, it's no showstopper)